package.json

```diff
@@ -8,8 +8,8 @@
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
     "clean": "rm -rf dist",
-    "build": "npm run clean && tsc",
-    "lint:fix": ""
+    "build": "npm run lint && npm run clean && tsc",
+    "lint": "tslint --project tsconfig.json protractor/**/*.ts test/**/*.ts src/**/*.ts"
```

Comment:
why do you add the protractor folder?
Is that linter working for cypress? What about using a cypress plugin instead?

Reply:
https://github.com/cypress-io/eslint-plugin-cypress

Reply:
I have added cypress folder to tslint target and removed the protractor entry, the linter seems to be doing ok with cypress, I don't think the plugin is needed.